src/components/Config/Config.tsx

```diff
@@ -120,68 +146,148 @@ function Config() {
+              Metamask extension.
+            </a>{' '}
+            If you have installed refresh the page.
+          </span>
+        ),
+      });
+    }
+  }, []);
+
   return (
     <Container>
       <PageTitle title="Connect" />
+
       <Typography.Title level={3} className="title">
-        {completed ? 'Congratulations!' : 'Connection Settings'}
+        {completed ? 'Congratulations!' : networkvalue < 2 ? 'Connection Settings' : 'Select Network'}
       </Typography.Title>
       {completed && (
         <Completed>
-          <p>You can now start saving on gas fees! Click one of the links below to proceed.</p>
+          <p>You can now start scheduling transactions! </p>
         </Completed>
       )}
       {!completed && (
         <>
-          <Checkboxes>
-            <CheckboxSection>
-              <Checkbox
-                checked={gasPriceAware}
-                disabled={submitted}
-                onChange={(e) => setGasPriceAware(e.target.checked)}
-              >
-                <Typography.Paragraph className="subtitle">Gas price aware</Typography.Paragraph>
-              </Checkbox>
-              <p>
-                Once the network gas price falls below the gas cost specified in the transaction, it will be broadcast
-                to the network.
-              </p>
-            </CheckboxSection>
-            <CheckboxSection>
-              <Checkbox checked={draft} disabled={submitted} onChange={(e) => setDraft(e.target.checked)}>
-                <Typography.Paragraph className="subtitle">Draft mode (Advanced)</Typography.Paragraph>
-              </Checkbox>
-              <p>
-                No transaction will be broadcast to the Ethereum network until you go to the Transaction list and
-                specify additional conditions.
-              </p>
-            </CheckboxSection>
-          </Checkboxes>
-          <div>
-            <Typography.Paragraph className="subtitle">Estimated Confirmation Time</Typography.Paragraph>
-            <p>
-              Our algorithms analyze historical gas prices in real time to best propose a gas price for you. Longer wait
-              times generally correspond to cheaper gas prices. Since we cannot control the Ethereum network, this is
-              NOT a guaranteed confirmation time.
-            </p>
-            <SliderContainer>
-              <Slider
-                marks={sliderMarks}
-                step={1}
-                value={confirmationTime}
-                min={0}
-                max={3}
-                disabled={submitted}
-                tooltipVisible={false}
-                onChange={setConfirmationTime}
-              />
-            </SliderContainer>
-          </div>
-          <Button type="primary" size="large" onClick={handleConnect}>
+          <Radio.Group
+            defaultValue={1}
+            onChange={(e) => setnetworkValue(e.target.value)}
```

Comment:
use proper casing - each word should be capitalized (except the first) so `setNetworkValue`